test/testdrive/transactions-timedomain-nonmaterialized.td

```diff
@@ -7,7 +7,7 @@
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0.
 
-# Test behavior of timedomains with non-materialized sources. Currently they prevent transactions
+# Test behavior of timedomains with non-materialized sources. Currently they prevent transactions.
```

Comment:
it seems like they no longer prevent transactions is that right?

Reply:
Fixed.